frontend/app/generateUserDetails.js

```diff
@@ -169,12 +171,14 @@ export function generateUserDetails() {
  */
 export function updateDropDown(req) {
     const request = JSON.parse(req.responseText);
-    const names = ['mortgage', 'unmortgage'];
-    const options = [request.mortgage, request.unmortgage];
+    console.log(`Response:${request}`);
+    const names = ['mortgage', 'unmortgage', 'properties-house'];
+    const options = [request.mortgage, request.unmortgage, request.mortgage];
```

Comment:
Should the last element in `options` be something other than `request.mortgage`? With that said, `property_state.py` isn't returning anything related to buying properties.

Reply:
It's returning the list of properties you can mortgage. Which is the same as the list of properties you can buy a house on right? As you can' t buy a house on a mortgaged property. I could be wrong here!

Reply:
Sorry, that's correct, that was just me forgetting the rules for a moment! Also I was just going to wonder why I wan't able to buy a house on a space, but I realised I have to buy the _property_ first. In that case, will you need to merge Noel's branches into this so we can buy properties?

Reply:
I realized over the course of the project I don't know the rules of the game myself. There's a bit of fear when I upload something like this that I've done it wrong altogether! :upside_down_face: I know it's a nightmare to test. I was running a little script on the database to buy Old Kent Road for testing! I'll let Noel merge his branch before I merge anything.